notes.md

```diff
@@ -101,19 +101,19 @@ The position in some Elements refers to the position, in octets, from the beginn
 
 The exact time of an object represented in nanoseconds. To find out a Block's Raw Timecode, you need the Block's timecode, the [Cluster's]({{site.baseurl}}/index.html#Cluster) [Timecode]({{site.baseurl}}/index.html#Timecode), and the TimecodeScale. For calculation, please see the see the <a href="notes.html#TimecodeScale">TimecodeScale notes.</a>
 
-### Segment linking
+### Linked Segments
 
-#### Hard linking
+#### Hard Linking
 
-This linking can also be called splitting. It's the operation of cutting one Segment in several parts. The resulting parts should play as if it was just one part (the original Segment). That means the timecode of each part follows the ones from the previous parts. The track numbers are the same. The chapters only match the current Segment (unless the edition is ordered, where all parts should be in each Segment). And most important, the NextUID and PrevUID points the respective Segment UIDs.
+Hard Linking (also called splitting) is the operation of cutting one Segment in several smaller linked Segments. The resulting Linked Segments should play as if it was the original Segment. Within a Linked Segment the timecode of each Segment MUST follow consequtively and the track numbers and timescale MUST be the same amongst the Segments. The chapters only match the current Segment (unless the Edition is ordered, where all parts should be in each Segment). Within a Linked Segment the NextUID and PrevUID MUST reference the respective Segment UIDs of the next and previous Segments.
```

Comment:
I'm going to add an intro paragraph to Linked Segments above Hard Linking, because some of the mandates in the Hard Linking section (track consistency) actually apply to each type of Linking.